flow-core/src/utils/rdfjs-utils.ts

```
@@ -26,7 +26,69 @@ export async function jsonldToQuads(
+
+export function relativizeQuads(inputQuads: RDF.Quad[], baseIRI: string): RDF.Quad[] {
+  // Validate baseIRI format
+  if (!baseIRI.startsWith("http://") && !baseIRI.startsWith("https://")) {
+    throw new Error(`Invalid baseIRI: must start with http:// or https://, got: ${baseIRI}`);
+  }
+  if (!baseIRI.endsWith("/")) {
+    throw new Error(`Invalid baseIRI: must end with "/", got: ${baseIRI}`);
+  }
+
+  return inputQuads.map(quad => {
+    const subject = quad.subject.termType === 'NamedNode' ? quad.subject.value.replace(baseIRI, '') : quad.subject.value;
```

ESLint

[error] Strings must use doublequote.

(quotes)